will/__init__.py

```diff
@@ -1 +1 @@
-VERSION = "2.1.3"
+VERSION = "4.0.0"
```

Comment:
I'd say we're due for a new release but not this major of one. Lets be conservative with 2.2.0 to mark a major rewrite of the slack backend. @skoczen then we can push a full release to pypi.